tests/e2e/skills/test_skills_llm_e2e.py

```
@@ -71,6 +71,13 @@
 _RESULTS: list[dict] = []
 
 
+def _normalize_hyphens(text: str) -> str:
+    """Replace Unicode dash variants with ASCII hyphen for robust assertions."""
+    for ch in "‐‑‒–—―−":
```

Ruff (0.15.18)

[warning] String contains ambiguous `‐` (HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)